css/style.css

```diff
@@ -206,3 +206,9 @@ nav ul li a:focus {
   background-color: #FFFFFF;
     border: 1px solid #D7E0E0;
 }
+
+/* Adding a new secondary color #555555 (Davy's Grey). It is used by map-block-time(timestamp) of beta.freecodecamp.org - By @vkWeb */
```

Comment:
Can you remove this comment? The credit will be saved in the git history. Not having this comment will save us the trouble of having to update it when things change (not new anymore, used in a different place, changed by somebody else).

Reply:
Sure :)